Source/JavaScriptCore/jit/RegisterSet.h

```diff
@@ -280,7 +280,13 @@ class RegisterSet final {
         auto effectiveGPRCount = numberOfSetFPRs()
             ? WTF::roundUpToMultipleOf<2>(numberOfSetGPRs())
             : numberOfSetGPRs();
-        return effectiveGPRCount * bytesForWidth(pointerWidth()) + numberOfSetFPRs() * sizeof(double);
+        // FIXME: is this correct?
```

Comment:
I think this could be better. m_bits * sizeof(CPURegister) + m_upperBits * sizeof(double) seems like what we want, right?